### PR TITLE
fix: transcribe verbosity

### PR DESCRIPTION
**Pull request by bottledmind**
I've hidden "Detected langauge: " massage behind `verbose` flag.
Also fixed the flag usage with tqdm

### CODE REVIEW

**Comment by jongwook** [comment]
I actually intended the two to be displayed even when `--verbose` is False, thinking it'd help the user notice any language detection errors and show the progress/ETA without flooding the console.

**Comment by bottledmind** [comment]
@jongwook well, ETA does not work when verbose=True, which is misleading as it seem to me

**Comment by jongwook** [comment]
The language tag is returned in the result, but for those calling `whisper` in the command line (as it seems to be a popular use case) the detail is lost.

Regarding ETA: that's true; it's not the most intuitively named options but I thought both formats have merits (like below) and thought it'd be nice to have these verbose/concise output options, but probably better not mix up those two.

image

**Comment by bottledmind** [comment]
How about str verbosity like "full", "minimal" and "silent"? I might add it real quick

**Comment by jongwook** [comment]
I'd like to avoid magic string if possible. It's a bit unorthodox but I'm inclined to making it an `Optional[bool]` and use `None`/`False`/`True` so `verbose=None` is the default in `transcribe()` but from the command line it selects either `True` or `False`.

I can make these edits on this PR if you'd like/don't mind.

**Comment by bottledmind** [comment]
Don't bother yourself, I have nothing to do at the moment

File: whisper/transcribe.py [hunk 1]
```diff
@@ -170,7 +172,7 @@ def add_segment(
     num_frames = mel.shape[-1]
     previous_seek_value = seek
 
-    with tqdm.tqdm(total=num_frames, unit='frames', disable=verbose) as pbar:
+    with tqdm.tqdm(total=num_frames, unit='frames', disable=verbose is False) as pbar:
```
Comment (jongwook) [inline whisper/transcribe.py:175]: Could you make this `verbose is not False`? Thank you.

Comment (bottledmind) [reply whisper/transcribe.py]: Done

**Pull request merged**